src/libs/actions/Report.js

```diff
@@ -313,11 +313,18 @@ function fetchChatReportsByIDs(chatList, shouldRedirectIfInacessible = false) {
             Log.info('[Report] successfully fetched report data', false, {chatList});
             fetchedReports = reportSummaryList;
 
-            // If we receive a 404 response while fetching a single report, treat that report as inacessible.
+            // If we receive a 404 response while fetching a single report, treat that report as inaccessible.
             if (jsonCode === 404 && shouldRedirectIfInacessible) {
                 throw new Error(CONST.REPORT.ERROR.INACCESSIBLE_REPORT);
             }
 
+            // If the user doesn't have a report with Concierge yet, we need to create it
+            const hasConciergeChat = _.some(reportSummaryList, report => isConciergeChatReport(report));
+            if (!hasConciergeChat) {
+                // eslint-disable-next-line no-use-before-define
+                fetchOrCreateChatReport([currentUserEmail, CONST.EMAIL.CONCIERGE], false);
```

Comment:
Hmm... sorry I'm not sure this is such a great idea. There are at least two cases I noticed where the `reportSummaryList` will not be an exhaustive list of every report someone has - so we are going to be trying to create this report for no reason in those cases. 

Can we think of something better?

Reply:
Ah, because fetchChatReportByIDs doesn't necessarily need to be called with all of a user's reports, got it. Looking into this now!

Reply:
@marcaaron Fixed it by moving the logic for Concierge report creation into `fetchAllReports`, let me know what you think!